benchmarks/glyco/build_entrap.py

```
@@ -0,0 +1,44 @@
+            if acc: yield acc, "".join(seq)
+            acc, seq = ln.rstrip("\n"), []
+        else:
+            seq.append(ln.strip())
+    if acc: yield acc, "".join(seq)
+
+n_m = n_e = 0
+with open(OUT, "w") as out:
+    for acc, seq in read_fasta(MOUSE):          # mouse target+decoy verbatim
+        out.write(f"{acc}\n{seq}\n"); n_m += 1
+    for acc, seq in read_fasta(HYE):
+        if "Cont_" in acc: continue
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)